### PR TITLE
Minikube debugging documentation

### PR DESCRIPTION
**Pull request by willemodendaal**
I ran into a few issues when installing Minikube locally. These docs should help other people who run into the same issues.

### CODE REVIEW

File: .vscode/launch.json [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+            }
+        },
+        {
+            "name": ".NET Core Attach",
+            "type": "coreclr",
+            "request": "attach",
+            "processId": "${command:pickProcess}"
+        }
+    ,]
+}
```
Comment (JPThorne) [inline .vscode/launch.json:46]: Okay so this file is a ... launch properties file for debugging from vscode?

File: .vscode/tasks.json [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+            "command": "dotnet",
+            "type": "process",
+            "args": [
+                "build",
+                "${workspaceFolder}/src/API/Loggy.Api/Loggy.Api.csproj"
+            ],
+            "problemMatcher": "$msCompile"
+        }
+    ]
+}
```
Comment (JPThorne) [inline .vscode/tasks.json:15]: Okay cool so is this used ... by minikube?

File: doc/minikube windows 10 debugging tips.md [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+To get past this error, do the following:
+
+- Open Hyper-V Manager.
+- Stop minikube virtual machine.
+- Delete minikube virtual machine.
+- Open Explorer and go to folder: C:\users\username\.minikube\machines
+- Delete minikube folder (this will delete the minikube virtual machine and associated configuration).
+- Open an elevated command prompt and start from scratch: `minikube start --vm-driver="hyperv" --hyperv-virtual-switch="minikubewifi" --v=9`
+
+This should re-create the Minikube virtual machine in a working state.
```
Comment (JPThorne) [inline doc/minikube windows 10 debugging tips.md:72]: +1 really nice and descriptive!

**Review by JPThorne — APPROVED** [review]
Neato! Wonder how it runs on windows 8 haha

File: .vscode/launch.json (hunk 1 above)
Comment (willemodendaal) [reply .vscode/launch.json:46]: Yeah, actually all auto-generated. I wasn't sure if .vscode/ should really be checked in or not, but I figured it was like solution settings that we all shared.

File: .vscode/tasks.json (hunk 2 above)
Comment (willemodendaal) [reply .vscode/tasks.json:15]: Nope, still just auto-generated VSCode things. Should really have gone in a separate PR.

File: doc/minikube windows 10 debugging tips.md (hunk 3 above)
Comment (willemodendaal) [reply doc/minikube windows 10 debugging tips.md:72]: 👍

**Pull request merged**